### PR TITLE
Checking if there are any releases associated with recording in _get_recording_info function

### PR DESCRIPTION
**Pull request by gentlecat**
*Fixes http://sentry.roman.zone/sentry/ab-production/group/30/ and http://tickets.musicbrainz.org/browse/AB-49.*

### CODE REVIEW

File: webserver/views/data.py [hunk 1]
```diff
@@ -152,12 +152,13 @@ def _get_recording_info(mbid, metadata):
         info['title'] = good_metadata['title']
         info['artist_id'] = good_metadata['artist-credit'][0]['artist']['id']
         info['artist'] = good_metadata['artist-credit-phrase']
-        info['release_id'] = good_metadata['release-list'][0]['id']
-        info['release'] = good_metadata['release-list'][0]['title']
-        info['track_id'] = good_metadata['release-list'][0]['medium-list'][0]['track-list'][0]['id']
-        info['track_number'] = \
-            '%s / %s' % (good_metadata['release-list'][0]['medium-list'][0]['track-list'][0]['number'],
-                         good_metadata['release-list'][0]['medium-list'][0]['track-count'])
+        if good_metadata['release-list']:
```
Comment (alastair) [inline webserver/views/data.py]: at the very least we should do `release = good_metadata['release-list'][0]`

Comment (gentlecat) [inline webserver/views/data.py]: *http://chatlogs.musicbrainz.org/musicbrainz-devel/2015/2015-08/2015-08-05.html#T21-44-38-154774*

Comment (gentlecat) [inline webserver/views/data.py]: Done in 5c29273c7bb9788bc1c3b5ae69d057a28254919d.

**Comment by alastair** [comment]
:ship: it

**Pull request merged**